### PR TITLE
Update readme with template vars and goreleaser fixes

### PR DESCRIPTION
**Pull request by nikkictl**
Signed-off-by: Nikki Attea <nikki@sensu.io>

### CODE REVIEW

File: .bonsai.yml [hunk 1]
```diff
@@ -41,14 +41,6 @@
...
     -  "entity.system.arch == 'amd64'"
   
-  - platform: "OSX"
-    arch: "386"
-    asset_filename: "#{repo}_#{version}_darwin_386.tar.gz"
-    sha_filename: "#{repo}_#{version}_sha512-checksums.txt"
-    filter:
-    -  "entity.system.os == 'darwin'"
-    -  "entity.system.arch == '386'"
-
```
Comment (ccressent) [inline .bonsai.yml:51]: There are Intel Macbooks running OSX with 32bit processors; are we sure we want to remove this combination?
See: https://support.apple.com/en-ca/HT201948

File: go.mod [hunk 2]
```diff
@@ -1,3 +1,8 @@
-module github.com/sensu-community/handler-plugin-template
+module github.com/{{ .GithubUser }}/{{ .GithubProject }}
 
 go 1.13
+
+require (
+	github.com/sensu-community/sensu-plugin-sdk v0.5.0-alpha3
```
Comment (ccressent) [inline go.mod]: There are newer versions of `sensu-plugin-sdk` available; should we update that too?

File: README.md [hunk 3]
```diff
@@ -1 +1,91 @@
...
+## Installation from source
+
+The preferred way of installing and deploying this plugin is to use it as an Asset. If you would
+like to compile and install the plugin from source or contribute to it, download the latest version
+or create an executable script from this source.
+
+From the local path of the {{ .GithubProject }} repository:
+
+```
+go build -o /usr/local/bin/{{ .GithubProject }} main.go
```
Comment (ccressent) [inline README.md]: We shouldn't output to a system directory like that: that forces `go build` to run as `root`, which it should never do. `go build` only builds, it doesn't install! People need to run `go install` if they want to "install" the resulting binary.

Also `go help build` mentions:
> If the arguments to build are a list of .go files from a single directory, build treats them as a list of source files specifying a single package.

I wonder if specifying `main.go` breaks the build if the `main` package is split across several files (I haven't tried).

I think we should replace that line with just `go build`, or _maybe_ `go build -o {{ .GithubProject }}` if we really want that particular name for the binary.

File: .bonsai.yml (hunk 1 above)
Comment (nikkictl) [reply .bonsai.yml:51]: https://github.com/sensu-community/handler-plugin-template/pull/4#discussion_r377097932

File: README.md (hunk 3 above)
Comment (nikkictl) [reply README.md]: Following the instructions from https://github.com/sensu-plugins/community/blob/master/PLUGIN_STYLEGUIDE.md#installation-from-source. If those instructions are incorrect, we should start there.

File: .bonsai.yml (hunk 1 above)
Comment (palourde) [reply .bonsai.yml:51]: 32-bit programs have been deprecated for a couple of macOS releases now and they are no longer supported on Catalina. We also only provide 64-bit versions of sensuctl & sensu-agent on our website so I think it makes total sense to remove those from the supported plugins!

File: README.md (hunk 3 above)
Comment (ccressent) [reply README.md]: I think we should nip this in the bud before people start (hopefully) using those templates, but I don't want to block this PR.

Here's the course of action I suggest: I'll open issues against the plugin style guide and the existing plugins that it references. If people agree with my proposed changes, I can then open appropriate PRs across the board.

**Review by ccressent — APPROVED** [review]

**Pull request merged**